### PR TITLE
Fix a bug in the tuffleHog output loading logic, and add "password" to rules.json

### PR DESCRIPTION
**Pull request by mhmdiaa**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -255,11 +255,11 @@ func cloneusergists(ctx context.Context, client *github.Client, user string) err
 		fmt.Println(gisturl)
 
 		//cloning the individual user gists
-		func(gisturl string, userGist *github.Gist, user string, usergistclone *sync.WaitGroup) {
```
Comment (anshumanbh) [inline main.go]: why did we remove this part?

File: main.go [hunk 2]
```diff
@@ -736,7 +751,7 @@ func checkflags(token string, org string, user string, repoURL string, gistURL s
 					os.Exit(2)
 				}
 			}
-		} else if org != "" && teamName == "" {
```
Comment (anshumanbh) [inline main.go]: why did we remove this part?

File: main.go (hunk 1 above)
Comment (mhmdiaa) [reply main.go]: Disregard this part; it was a mistake. The only relevant part is 533:565. I committed the modified parts back in.

File: main.go (hunk 2 above)
Comment (mhmdiaa) [reply main.go]: Same as above.

**Comment by anshumanbh** [comment]
I had already made these changes in the last push I did to the master. So, will close this!

**Pull request closed without merging**